tests/lib/rules/no-triple-slash-reference.js

```diff
@@ -39,6 +39,17 @@ ruleTester.run("no-triple-slash-reference", rule, {
                     column: 1
                 }
             ]
+        },
+        {
+            code: '/// <reference path="Animal" />\nlet a',
```

Comment:
This fails with `getComments(program).leading` because `leading` seems empty.

Reply:
Can we use a template literal instead of the \n?

```javascript
code: `
/// <reference path="Animal" />
let a;
`
```

I think it makes it more readable.

Reply:
Sure. Tomorrow.

Reply:
Actually, this seems to result in a different string, because the leading whitespaces and the extra newlines are not trimmed. There are template tags like [this one](https://www.npmjs.com/package/common-tags#stripindents) but seems overkill to me.

Reply:
You can move the code all the way to the left, check how we do it in other tests, most of them use this approach

Reply:
I agree adding the template would be an overkill, that's why we just ignore the leading spaces by removing the indentation

Reply:
By the way, I created bug #81 to standardize the tests using this same approach and some other suggestions made by @j-f1... Currently working on this...